run/run_test.go

```diff
@@ -28,6 +29,18 @@ func TestRun(t *testing.T) {
 		t.Fatalf("failed to initialize a testing option: %v", err)
 	}
 
+	var jourctlOpt *option.Option
+	var jerr error
+	if *subject == "finch" {
+		jourctlOpt, jerr = option.New(
+			[]string{"/Applications/Finch/lima/bin/limactl", "shell", "finch", "sudo", "journalctl"},
+			option.Env(append(os.Environ(), "LIMA_HOME=/Applications/Finch/lima/data")),
```

Comment:
`NewCmd` is already appending `os.Environ()`, so we may not need to append it here.

Reply:
Can we make a sudoOpt instead of jourctlOpt here? There are other commands we need to execute when testing other flags like --security-opt, e.g. `mknod`. I feel making a `sudoOpt` suits more use cases. WDYT?
@davidhsingyuchen

Reply:
I'm bit hesitant about this because `sudo` is sort of an implementation detail. For example, we're still not sure why we need to add `sudo` to `journalctl` to see the logs, and maybe we can fix it in the future and drop the `sudo`. Also, the responsibility of `sudo` may become a bit vague. For example, we'll need to somehow translate "// JournalctlOpt instructs how to run the journalctl binary. If it's nil, test cases related to `--log-driver journald` will be skipped." to probably keep track of the commands that rely on `sudo` and the corresponding test cases.